compiler/env/FEBase_t.hpp

```diff
@@ -50,19 +50,20 @@ uint8_t *
 FEBase<Derived>::allocateCodeMemory(TR::Compilation *comp, uint32_t warmCodeSize, uint32_t coldCodeSize,
                             uint8_t **coldCode, bool isMethodHeaderNeeded)
    {
-   TR::CodeCache *codeCache = static_cast<TR::CodeCache *>(comp->getCurrentCodeCache());
+   TR::CodeGenerator *cg = comp->cg();
+   TR::CodeCache *codeCache = static_cast<TR::CodeCache *>(cg->getCodeCache());
```

Comment:
I don't think this `static_cast` is needed because we already get a `TR::CodeCache *`:

```suggestion
   TR::CodeCache *codeCache = cg->getCodeCache();
```